### PR TITLE
fix: UI build error & Ci job for build check

### PR DESCRIPTION
**Pull request by kushthedude**
**Notes to reviewers**

#### Checklist

- [x] [Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin).
- [x] PR title includes `[<name of area>]:<description>`

### CODE REVIEW

**Comment by kushthedude** [comment]
@leecalcote Ready to be merged!

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Meshery
 mesheryctl/cmd/mesheryctl/mesheryctl
 mesheryctl/.meshery/
+mesheryctl/mesheryctl
```
Comment (leecalcote) [inline .gitignore:4]: Is this right?

Comment (kushthedude) [reply .gitignore:4]: It was created when I built mesheryctl using `makefile`

Comment (kushthedude) [reply .gitignore:4]: Ah, got it. This is the previous built binary before @nitishm introduced a change in structure

Comment (kushthedude) [reply .gitignore:4]: Fixed!

**Pull request merged**